core/ledger/kvledger/txmgmt/privacyenabledstate/snapshot_test.go

```diff
@@ -363,6 +363,9 @@ func TestSnapshotReaderNextFunction(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, "", retrievedNs)
 	require.Nil(t, retrievedSr)
+
+	_, err = newSnapshotReader("test-dir", "test-data-filename", "test-metadata-filename")
+	require.EqualError(t, err, "file does not exist")
```

Comment:
This test is misplaced, under `TestSnapshotReaderNextFunction`. Moving this under `TestSnapshotImportErrorPropagation` makes a better fit.